primitives/Cargo.toml

```diff
@@ -1,49 +1,49 @@
-license = "Apache-2.0"
-categories = ["cryptography::cryptocurrencies"]
 keywords = ["nimiq", "cryptocurrency", "blockchain"]
+license = "Apache-2.0"
+name = "nimiq-primitives"
+repository = "https://github.com/nimiq/core-rs-albatross"
+version = "0.1.0"
 
 [badges]
-travis-ci = { repository = "nimiq/core-rs", branch = "master" }
-is-it-maintained-issue-resolution = { repository = "nimiq/core-rs" }
-is-it-maintained-open-issues = { repository = "nimiq/core-rs" }
-maintenance = { status = "experimental" }
+is-it-maintained-issue-resolution = {repository = "nimiq/core-rs"}
+is-it-maintained-open-issues = {repository = "nimiq/core-rs"}
+maintenance = {status = "experimental"}
+travis-ci = {repository = "nimiq/core-rs", branch = "master"}
 
 [dependencies]
 bitvec = "0.17"
-thiserror = { version = "1.0.20", optional = true }
-hex = { version = "0.4", optional = true }
-itertools = { version = "0.10", optional = true }
-lazy_static = { version = "1.2", optional = true }
+hex = {version = "0.4", optional = true}
+itertools = {version = "0.10", optional = true}
+lazy_static = {version = "1.2", optional = true}
 log = "0.4"
-num-bigint = { version = "0.4.2", optional = true }
-num-traits = { version = "0.2", optional = true }
-parking_lot = { version = "0.11", optional = true }
-regex = { version = "1.3", optional = true }
-serde = { version = "1.0", features = ["derive"], optional = true }
+num-bigint = {version = "0.4.2", optional = true}
+num-traits = {version = "0.2", optional = true}
+parking_lot = {version = "0.11", optional = true}
+regex = {version = "1.3", optional = true}
+serde = {version = "1.0", features = ["derive"], optional = true}
 strum_macros = "0.22"
+thiserror = {version = "1.0.20", optional = true}
 
-beserial = { path = "../beserial" }
-beserial_derive = { path = "../beserial/beserial_derive" }
-nimiq-bls = { path = "../bls", features = ["beserial"], optional = true }
-nimiq-keys = { path = "../keys",  optional = true }
-nimiq-utils = { path = "../utils", features = ["math"], optional = true}
+beserial = {path = "../beserial"}
+beserial_derive = {path = "../beserial/beserial_derive"}
+nimiq-bls = {path = "../bls", features = ["beserial"], optional = true}
+nimiq-keys = {path = "../keys", optional = true}
+nimiq-utils = {path = "../utils", features = ["math"], optional = true}
 
 [dev-dependencies]
 lazy_static = "1.2"
 
 [features]
-all = ["coin", "account", "policy", "networks", "slots"]
-coin = ["hex", "lazy_static", "thiserror", "num-traits", "regex"]
 account = ["hex", "thiserror"]
-policy = ["num-bigint", "num-traits", "parking_lot", "lazy_static"]
+all = ["account", "coin", "networks", "policy", "slots"]
+coin = ["hex", "lazy_static", "num-traits", "regex", "thiserror"]
 networks = ["thiserror"]
-slots = ["nimiq-bls", "nimiq-utils","nimiq-keys", "beserial/bitvec", "itertools", "policy"]
+policy = ["lazy_static", "num-bigint", "nimiq-keys", "num-traits", "parking_lot"]
```

Comment:
Most of the changes are cosmetic except for adding `nimiq-keys` in the `policy` feature.